apps/src/templates/instructions/TeacherFeedback.jsx

```diff
@@ -50,7 +50,9 @@ class TeacherFeedback extends Component {
   constructor(props) {
     super(props);
     const search = window.location.search;
-    const studentId = search.split('&')[1].split("=")[1];
+    //Pull the student id from the url
+    let studentId = "";
+    search.split('&').map(item => { studentId = item.includes('user_id') ? item.split('=')[1] : studentId; });
```

Comment:
A couple nits here:
1. Since we only care about one of the items I find `map` less intuitive than `find` here.
2. This doesn't check word boundaries; suppose in the future we added `pair_user_id` to the querystring for this page, this code would be ambiguous about which id we were retrieving.

I have two suggestions.  The first is to use the `query-string` package we already depend on.
```js
import queryString from 'query-string';
const studentId = queryString.parse(window.location.search).user_id;
```
The second, if you prefer not to depend on that library, would be to use `Array.protoytpe.find` and RegExp to locate and extract the value.
```js
const studentId = search.split('&').find((pair) => {
  const match = pair.match(/^user_id=(.*)$/);
  return match && decodeUriComponent(match[1]);
});
```
Using the library is probably the more dependable way to go.